src/epics/revisions.js

```diff
@@ -19,11 +19,23 @@ const buildUrl = ( state ) => {
 	return url;
 };
 
-const fetchRevisions = ( action$, store ) => (
+export const showLoader = ( action$, store ) => (
```

Comment:
Rename this to something more descriptive... maybe `shouldFetchRevisions` ?

Reply:
Done